src/zenml/integrations/slack/alerters/slack_alerter.py

```diff
@@ -131,6 +141,59 @@ def _get_disapprove_msg_options(
             return params.disapprove_msg_options
         return DEFAULT_DISAPPROVE_MSG_OPTIONS
 
+    def _create_blocks(
+        self, message: str, params: Optional[BaseAlerterStepParameters]
+    ) -> List[Dict]:  # type: ignore
+        """Define which messages will lead to disapproval during ask().
+
+        Args:
+            message: message
+            params: Optional parameters.
+
+        Returns:
+            Set of messages that lead to disapproval in alerter.ask().
```

Comment:
As far as I can see, this block creation has nothing to do with disapproval during `ask()`. The docstring probably needs updating.

Reply:
done !